spec/recipes/omnibus_toolchain_spec.rb

```diff
@@ -1,11 +1,17 @@
 require 'spec_helper'
 
 describe 'omnibus::_omnibus_toolchain' do
-  let(:chef_run) { ChefSpec::SoloRunner.converge(described_recipe) }
+  let(:chef_run) {
+    ChefSpec::SoloRunner.new do|node|
+      node.automatic['kernel']['machine'] = 'x86_64'
+      node.automatic['kernel']['bits'] = '64'
+      node.automatic['architecture'] = 'x86_64'
```

Comment:
node.automatic['kernel']['bits'] = '64'
node.automatic['architecture'] = 'x86_64'

can be removed